metadrive/component/sensors/instance_camera.py

```diff
@@ -0,0 +1,48 @@
+from metadrive.component.sensors.semantic_camera import SemanticCamera
+import cv2
+from panda3d.core import GeoMipTerrain, PNMImage
+from panda3d.core import RenderState, LightAttrib, ColorAttrib, ShaderAttrib, TextureAttrib, LVecBase4, MaterialAttrib
+from metadrive.constants import Semantics
+from metadrive.component.sensors.base_camera import BaseCamera
+from metadrive.constants import CamMask
+from metadrive.constants import RENDER_MODE_NONE
+from metadrive.engine.asset_loader import AssetLoader
+from metadrive.engine.engine_utils import get_engine
+import random
+
+
+class InstanceCamera(SemanticCamera):
```

Comment:
We can have a test for InstanceCamera. A good reference is `tests/test_sensors/test_rgb_camera`. You can copy that file, rename related text and replace the RGBCam with InstanceCam. It can at least ensure the sensor pipeline works well like creating InstanceCamera, and capturing image observations. 

If you have some method to check the returned image is as your expectation, you can also add these checks to the code to make the test better. For example, the pixel at row 61, column 52 should be in green, while the returned value is black. In this case we can throw an error